.github/workflows/concrete_python_release.yml

```diff
@@ -86,6 +86,8 @@ jobs:
             ${{ env.DOCKER_GPU_OPTION }}
           shell: bash
           run: |
+            set -e
```

Comment:
wonder if it should not be put in the CP makefile (as it's the make whl that continues even if it encountered an error)

Reply:
I don't think it's the Makefile. The make command is failing properly in the CI AFAICT. It's the following commands that are executed and generating outputs. You should note that there is three make commands, the second one build the compiler python bindings is the one failing.